### ISSUE TITLE
Proof-read conversation and quest logs

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#99]
```
I'm not a native English speaker (I'm Swedish), so I might have missed some 
things or used strange wording when writing the conversation and quest texts. 

It would be great if someone that speaks English as mother-tongue could 
proof-read the texts and spot any mistakes in them. 

All conversations are stored in this file:
http://code.google.com/p/andors-trail/source/browse/trunk/AndorsTrail/res/values
/conversationlist.xml

All quest logs are stored in this file:
http://code.google.com/p/andors-trail/source/browse/trunk/AndorsTrail/res/values
/questlog.xml

There are also additional strings used in the UI here:
http://code.google.com/p/andors-trail/source/browse/trunk/AndorsTrail/res/values
/strings.xml


To read the conversation list in a more readable format, you can use the 
conversation editor from this url (copy the serialized contents into the lower 
textbox): 
http://andors-trail.googlecode.com/svn-history/r22/trunk/AndorsTrailEdit/editdia
log.html

```

Original issue reported on code.google.com by `oskar.wi...@gmail.com` on 20 Dec 2010 at 9:45

**Comment by GoogleCodeExporter** [comment]
```
Emailed fixes, too much to post here.
```

Original comment by `sdeva...@gmail.com` on 23 Dec 2010 at 5:50

**Comment by GoogleCodeExporter** [comment]
```
This is now updated. Thank you sdevaney for the input!
```

Original comment by `oskar.wi...@gmail.com` on 9 Jan 2011 at 6:29
- Changed state: **Ready**
- Added labels: **Milestone-0.6.8**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.8 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 24 Feb 2011 at 9:34
- Changed state: **Fixed**
